tutorials/pages/bridge-usdc-testnet-tokens.md

```diff
@@ -0,0 +1,159 @@
+# Bridge USDC Testnet Tokens using Multichain
+
+:::info
+
+This tutorial provides a step-by-step guide on how to use the Multichain testnet token bridge to bridge USDC Testnet tokens between the blockchains Goerli (Ethereum Testnetwork) and ShimmerEVM Testnet.
+
+:::
+
+## Prerequisites
+
+- [Metamask](https://metamask.io/) browser wallet with account
+
+## Networks and Tokens
+
+Before looking into the bridging process, we will make sure everything is set up so you can move USDC tokens between Goerli and ShimmerEVM Testnet in both directions. For this purpose you will need to add both networks to your Metamask wallet and fund your account with their respective base tokens to cover the gas fees that will occur. Last but not least you will get USDC Tokens on Goerli and add the USDC token contract for both Goerli and ShimmerEVM Testnet to your Metamask wallet.
+
+### Add networks to your Metamask wallet
+
+In order to get the needed networks to your Metamask wallet, you can use [Chainlist](https://chainlist.org/?testnets=true). Chainlist provides a comprehensive list of EVM networks, and easy to use shortcuts to add them to Metamask. Make sure the the box to _Include Testnets_ is checked, filter for `Goerli` and add the network to Metmask. Repeat this step for `ShimmerEVM Testnet`.
+
+<p style="text-align: center">
+    <kbd>
+        <img src="/static/search-goerli.png"  width="500">
+    </kbd>
+</p>
```

Comment:
Why did you choose to do it this way?
If there is a strong argument for this JSX syntax than at least fix it and remove the kbd tags:
```suggestion
<p style={{textAlign: center}}>
    <img src="/search-goerli.png"  width="500">
</p>
```

But I would actually prefer just using plaing markdown like:
`![ALT_TEXT](LINK_TO_IMAGE "Image Title")`
```suggestion
`![goerli-search](/search-goerli.png "Search Goerli")`
```

Can you please update all images? And please at least try to run the wiki first to check if your PR works